include/multigrid_pclomp/multigrid_ndt_omp.h

```diff
@@ -262,6 +262,8 @@ class MultiGridNormalDistributionsTransform : public pcl::Registration<PointSour
   // lower is better
   double calculateTransformationProbability(const PointCloudSource & cloud) const;
   double calculateNearestVoxelTransformationLikelihood(const PointCloudSource & cloud) const;
+  pcl::PointCloud<pcl::PointXYZI>::Ptr calculateNearestVoxelScoreEachPoint(
```

Comment:
Could you change the return type from `pcl::PointCloud<pcl::PointXYZI>::Ptr` to `pcl::PointCloud<pcl::PointXYZI>`?
Please make the same changes to `multigrid_ndt_omp_impl.hpp`.

Reply:
OK. I fixed it.